app/src/main/java/org/oppia/app/databinding/TextViewBindingAdapters.java

```diff
@@ -45,54 +45,56 @@ public static void setProfileLastVisitedText(@NonNull TextView textView, long ti
     );
   }
 
-  private static int SECOND_MILLIS = 1000;
-  private static int MINUTE_MILLIS = 60 * SECOND_MILLIS;
-  private static int HOUR_MILLIS = 60 * MINUTE_MILLIS;
-  private static int DAY_MILLIS = 24 * HOUR_MILLIS;
-
-  public static String getTimeAgo(long lastVisitedTimeStamp, Context context) {
-
+  private static String getTimeAgo(long lastVisitedTimeStamp, Context context) {
     OppiaDateTimeFormatter oppiaDateTimeFormatter = new OppiaDateTimeFormatter();
-    long timeStamp =
+    long timeStampMillis =
         oppiaDateTimeFormatter.checkAndConvertTimestampToMilliseconds(lastVisitedTimeStamp);
-    long now = oppiaDateTimeFormatter.currentDate().getTime();
+    long currentTimeMillis = oppiaDateTimeFormatter.currentDate().getTime();
 
-    if (timeStamp > now || timeStamp <= 0) {
+    if (timeStampMillis > currentTimeMillis || timeStampMillis <= 0) {
       return "";
     }
 
     Resources res = context.getResources();
-    long timeDifference = now - timeStamp;
+    long timeDifferenceMillis = currentTimeMillis - timeStampMillis;
 
-    if (timeDifference < MINUTE_MILLIS) {
+    if (timeDifferenceMillis < MINUTE_MILLIS) {
       return context.getString(R.string.just_now);
-    } else if (timeDifference < 50 * MINUTE_MILLIS) {
-      return context.getString(
-          R.string.time_ago,
-          res.getQuantityString(
-              R.plurals.minutes,
-              (int) timeDifference / MINUTE_MILLIS,
-              timeDifference / MINUTE_MILLIS
-          )
+    } else if (timeDifferenceMillis < 50 * MINUTE_MILLIS) {
+      return getPluralString(
+          context,
+          R.plurals.minutes,
+          (int) (timeDifferenceMillis / MINUTE_MILLIS)
       );
-    } else if (timeDifference < 24 * HOUR_MILLIS) {
-      return context.getString(
-          R.string.time_ago,
-          res.getQuantityString(
-              R.plurals.hours,
-              (int) timeDifference / HOUR_MILLIS,
-              timeDifference / HOUR_MILLIS
-          )
+    } else if (timeDifferenceMillis < 24 * HOUR_MILLIS) {
+      return getPluralString(
+          context,
+          R.plurals.hours,
+          (int) (timeDifferenceMillis / HOUR_MILLIS)
       );
-    } else if (timeDifference < 48 * HOUR_MILLIS) {
+    } else if (timeDifferenceMillis < 48 * HOUR_MILLIS) {
```

Comment:
As mentioned earlier, let's use TimeUnit.DAYS.toMillis(2) here instead of the arithmetic. Ditto above for the other 2 conditionals.

Reply:
Fixed!